src/app/swiper/keyboard/keyboard-swiper.tab.ts

```
@@ -0,0 +1,31 @@
+    selector: 'app-keyboard-swiper',
+    imports: [
+        JsonPipe,
+        SwiperComponent,
+        SwiperItemDirective
+    ],
+    templateUrl: './keyboard-swiper.tab.html',
+    styleUrl: './keyboard-swiper.tab.scss',
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class KeyboardSwiperTab {
```

ESLint

[error] Component class names should end with one of these suffixes: "Component" (https://angular.dev/style-guide#style-02-03)

(@angular-eslint/component-class-suffix)